src/Libraries/DSIronPython/IronPythonEvaluator.cs

```diff
@@ -148,6 +152,32 @@ private static string PythonStandardLibPath()
             return OutputMarshaler.Marshal(result);
         }
 
+        /// <summary>
+        /// Processes additional bindings that are not actual inputs.
+        /// Currently, only the node name is received in this way.
+        /// </summary>
+        /// <param name="scope">Python scope where execution will occur</param>
+        /// <param name="bindingNames">List of binding names received for evaluation</param>
+        /// <param name="bindingValues">List of binding values received for evaluation</param>
+        private static void ProcessAdditionalBindings(ScriptScope scope, IList bindingNames, IList bindingValues)
```

Comment:
is this code exactly the same? Does it make sense to add it to PythonNodeModel? I guess it's small enough the duplication is ok.

Reply:
ah, I see it's a `scriptScope`

Reply:
Not exactly the same. Adding something to the scope works different for each engine.